<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Dashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="uploadButton">
      <property name="text">
       <string>Upload PDF</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="createFolderButton">
      <property name="text">
       <string>Create Folder</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTreeWidget" name="fileTree"/>
    </item>
    <item>
     <widget class="QLineEdit" name="usernameInput">
      <property name="placeholderText">
       <string>New User Username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordInput">
      <property name="placeholderText">
       <string>New User Password</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addUserButton">
      <property name="text">
       <string>Add User</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="newUsernameInput">
      <property name="placeholderText">
       <string>New Admin Username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="newPasswordInput">
      <property name="placeholderText">
       <string>New Admin Password</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="updateCredentialsButton">
      <property name="text">
       <string>Update Admin Credentials</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="logoutButton">
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>